README.md

```diff
@@ -307,7 +307,8 @@ Currently, the configuration supports
   - `label`
   - `radio` (`div` wrapping radio button groups)
   - `checkbox` (`div` wrapping checkboxes)
-  - `help` (`span` containing help texts and validation messages)
+  - `help` (`span` containing validation messages)
+  - `hint` (`p` containing form input hints. Always visible by default; to hide hints until their form inputs receive focus, provide the same class name here as for `help`) 
```

Comment:
Are you sure about the last part? I tried this in the dummy app, and the hint was always visible.